README.md

```diff
@@ -1,38 +1,97 @@
+
+---
+
+###Technologies
+:computer: The technologies chosen for this project are:
+
+- <img src="https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png" alt="descrição da imagem" width="13"> Typescript
+- <img src="https://cdn-icons-png.flaticon.com/512/875/875209.png" alt="descrição da imagem" width="13"> React
+- :zap: ChakraUI
+- <img src="https://cdn.worldvectorlogo.com/logos/next-js.svg" alt="descrição da imagem" width="13"> Next
+
+---
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+###How to contribute
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+:memo: If you want to [contribute](https://github.com/joao-pedrozo/shoppingify-client/wiki/How-to-contribute%3F "Contribute") to the project, follow these step-by-step instructions to open a pull request.  
+:question: If you have any questions or issues, you can contact us on [Discord](https://t.co/QWAGh2nd9j).
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+---
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+###How to set up the project
+This is a Next.js project bootstrapped with create-next-app.
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+#####To get started, clone this repository using Git:
 
-## Learn More
+```bash
+# Clone this repository
+$ git clone https://github.com/your-user-name/your-project-name
+```
 
-To learn more about Next.js, take a look at the following resources:
+#####Then, install the project dependencies:
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+```bash
+# Install dependencies
+$ npm install
+```
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+#####Here are the available npm scripts:
 
-## Deploy on Vercel
+```bash
+#npm run dev
+dev: "next dev"
+
+#npm run build
+build: "next build"
+
+#npm start
+start: "next start"
+
+#npm run lint
+lint: "next lint"
+```
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+---
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+###Built With
```

Comment:
I think we don't need this bottom section since we are already showing what are the technologies in the section above. Just one thing that I would keep is this node version, but I think will be better if I showed it as a requirement instead of a technology used in the project.

Example: 
![image](https://user-images.githubusercontent.com/40037644/226759056-13b5270c-4104-4dab-822a-dde5d6e3e08e.png)

Reply:
You can replace this section with the MIT license section